### DISCUSSION TITLE
blogs/view/creating-a-gradient-reading-progress-bar-in-NextJS-and-tailwindcss

### DISCUSSION THREAD

**Discussion by giscus** [biplobsd/biplobsd.github.io#11] [Announcements]
# blogs/view/creating-a-gradient-reading-progress-bar-in-NextJS-and-tailwindcss

I am building this portfolio website to showcase my app and services to the world. During my researc... | Blogs | Biplob Sutradhar

https://biplobsd.me/blogs/view/creating-a-gradient-reading-progress-bar-in-NextJS-and-tailwindcss.md

**Comment by xementor** [comment]
thanks,  its very helpfull.

**Reply by biplobsd** [reply]
Thanks @xementor 🚀
